mysql-test/suite/innodb/t/insert_into_empty.test

```diff
@@ -405,4 +405,24 @@ CHECK TABLE t1;
 DROP TABLE t1;
 SET GLOBAL INNODB_DEFAULT_ROW_FORMAT= @format;
 
+--echo #
+--echo #   MDEV-30321 blob data corrupted by row_merge_write_blob_to_tmp_file()
+--echo #
+
+SET UNIQUE_CHECKS=0;
```

Comment:
These statements are redundant

Reply:
okay, good to know, thank you.
I has already about to remove them, but in another test, a few lines above, they are present, too, so I kept them

Reply:
You are right, the idea of this test file is that the `set` statement on the very first line covers all subsequent tests. The test for MDEV-27214 should not have added those redundant statements either. That is something that needs to be cleaned up separately, in the 10.6 branch. I will take care of that.

Reply:
The older test was cleaned up in fe38d7cad4fab33beba90eefaea9e9d4aef06a7c but not yet merged to 10.7 or later branches.

Reply:
thank you. I have adjusted my pr.